app/lib/efile/id/id_39r_calculator.rb

```diff
@@ -1,29 +0,0 @@
-module Efile
-  module Id
-    class Id39rCalculator < ::Efile::TaxCalculator
-      attr_reader :lines
-
-      def initialize(value_access_tracker:, lines:, intake:, year:)
-        super(intake: intake, year: year)
-
-        @value_access_tracker = value_access_tracker
-        @lines = lines
-        @intake = intake
-        @direct_file_data = intake.direct_file_data
-      end
-
-      def calculate
-        set_line(:ID39R_B_LINE_6, :calculate_line_b_6)
-      end
-
-      def calculate_line_b_6
-        [
-          @direct_file_data.total_qualifying_dependent_care_expenses,
-          [12_000 - @direct_file_data.excluded_benefits_amount, 0].max,
-          @direct_file_data.primary_earned_income_amount,
-          @direct_file_data.spouse_earned_income_amount,
-        ].min
-      end
-    end
-  end
-end
```

Comment:
why was this deleted?
 maybe you need to merge main? b/c i see commits made by me on this PR

Reply:
This was an extra id39r calculator class that was left from a merge before. Kevin and I discussed this and should be deleted